addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2791,37 +2791,6 @@ options.registry.ScrollButton = options.Class.extend({
     },
 });
 
-/**
- * Allows for images to be replaced.
- */
-options.registry.ReplaceImage = options.Class.extend({
```

Comment:
I would actually have removed that in 14.3 "while there is still time" (like I did for other editor dead code) but as you wish

Reply:
Moved to https://github.com/odoo/odoo/pull/70587/files